Source/Diagnostics/Diagnostics.cpp

```diff
@@ -575,6 +583,15 @@ Diagnostics::ComputeAndPack ()
             // Check that the proper number of components of mf_avg were updated.
             AMREX_ALWAYS_ASSERT( icomp_dst == m_varnames.size() );
 
+            if (m_diag_type == DiagTypes::TimeAveraged) {
+
+                const amrex::Real real_a = 1.0;
+                // call amrex sax operation to do the following
+                amrex::MultiFab::Saxpy(
+                        m_sum_mf_output[i_buffer][lev], real_a, m_mf_output[i_buffer][lev],
+                        0, 0, m_mf_output[i_buffer][lev].nComp(), m_mf_output[i_buffer][lev].nGrowVect());
```

Comment:
We could maybe write what AMReX's `Saxpy` does, as the name isn't that self-explanatory (is it?):
```suggestion
                // Compute m_sum_mf_output += real_a*m_mf_output
                amrex::MultiFab::Saxpy(
                        m_sum_mf_output[i_buffer][lev], real_a, m_mf_output[i_buffer][lev],
                        0, 0, m_mf_output[i_buffer][lev].nComp(), m_mf_output[i_buffer][lev].nGrowVect());
```